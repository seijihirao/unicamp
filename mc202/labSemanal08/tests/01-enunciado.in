inserir 50
inserir 30
inserir 40
inserir 10
inserir 20
pre-ordem
excluir 50
pre-ordem
terminar
