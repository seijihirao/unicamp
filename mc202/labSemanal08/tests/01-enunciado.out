50 30 10 20 40 
30 10 20 40 
